### PR TITLE
chore: update m365 check tenant message and add local config when not exist

### PR DESCRIPTION
**Pull request by xiaolang124**
1.Update check m365 tenant message to support unify config.
2.Add local config when not exist.

### CODE REVIEW

**Comment by xiaolang124** [comment]
Fix:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14142176
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14171519

File: packages/fx-core/src/core/environment.ts [hunk 1]
```diff
@@ -82,9 +86,28 @@ class EnvironmentManager {
     }
 
     envName = envName ?? this.getDefaultEnvName();
-    const configResult = await this.loadEnvConfig(projectPath, envName);
+    let configResult = await this.loadEnvConfig(projectPath, envName);
     if (configResult.isErr()) {
-      return err(configResult.error);
+      if (envName === "local") {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/core/environment.ts:91]: Maybe only generate the local config file when it doesn't exist, or user customized configs will be lost.

Comment (xiaolang124) [reply packages/fx-core/src/core/environment.ts]: Move to check local config file location.

**Review by HuihuiWu-Microsoft — DISMISSED** [review]
LGTM

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**